ansible/roles/debops.apt/defaults/main.yml

```diff
@@ -355,6 +355,19 @@ apt__original_sources:
                              ansible_local.apt.original_mirrors_deb|d())
                          else "absent" }}'
 
+                                                                   # ]]]
+# .. envvar:: apt__default_sources_present [[[
+#
+# Define if the default APT package sources for specific distributions should
+# be used or not.
+# If package sources are defined in specific group, host or on all hosts in
+# Ansible inventory, the default sources will be disabled.
+apt__default_sources_present: '{{ "True"
+                                  if ( not apt__group_sources and
+                                       not apt__host_sources and
+                                       not apt__sources )
+                                    else "False" }}'
```

Comment:
Yes, I agree. It's probably a disservice putting that logic there. 

However, I'm not sure I agree regarding changing the state-values from bool to `present`/`absent`, because it don't actually change the '`item.state`'. See https://github.com/debops/debops/pull/572#discussion_r240175949.

Reply:
I was aiming more for similarity with `apt__security_sources_state`, but if you think the functionality is different, fair enough. In that case, I would suggest a rename to `apt__default_sources_enabled` which, I think, more adequately describes this variable purpose, `_enabled` is also consistenly used in other DebOps roles for boolean variables that enable/disable parts of the role functionality without changing the state on the managed hosts.